concoction-lib/src/main/java/info/mmpa/concoction/scan/model/dynamic/entry/MethodLocation.java

```diff
@@ -0,0 +1,141 @@
+package info.mmpa.concoction.scan.model.dynamic.entry;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import info.mmpa.concoction.scan.dynamic.CallStackFrame;
+import info.mmpa.concoction.scan.model.TextMatchMode;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Outline of a method location.
+ */
+@JsonDeserialize(using = MethodLocationDeserializer.class)
+@JsonSerialize(using = MethodLocationSerializer.class)
+public class MethodLocation {
+	private final String className;
+	private final String methodName;
+	private final String methodDesc;
+	private final TextMatchMode classMatchMode;
+	private final TextMatchMode methodNameMatchMode;
+	private final TextMatchMode methodDescMatchMode;
+
+	/**
+	 * @param className
+	 * 		Declaring class of method.
+	 * @param methodName
+	 * 		Method name.
+	 * @param methodDesc
+	 * 		Method descriptor.
+	 * @param classMatchMode
+	 * 		Class name text matching technique.
+	 * @param methodNameMatchMode
+	 * 		Method name text matching technique.
+	 * @param methodDescMatchMode
+	 * 		Method descriptor text matching technique.
+	 */
+	public MethodLocation(@Nonnull String className,
+						  @Nonnull String methodName,
+						  @Nonnull String methodDesc,
+						  @Nonnull TextMatchMode classMatchMode,
+						  @Nonnull TextMatchMode methodNameMatchMode,
+						  @Nonnull TextMatchMode methodDescMatchMode) {
+		this.className = className;
+		this.methodName = methodName;
+		this.methodDesc = methodDesc;
+		this.classMatchMode = classMatchMode;
+		this.methodNameMatchMode = methodNameMatchMode;
+		this.methodDescMatchMode = methodDescMatchMode;
+	}
+
+	/**
+	 * @param frame
+	 * 		Frame to check.
```

Comment:
Space in javadoc `@param`

Reply:
?

Reply:
On my screen, theres a space between
```
@param name
```
and
`Frame to check.`

Reply:
This is how everything else in the project is formatted. Its consistent, and IMO vastly improves legibility in non-rendered Javadoc.